pkg/test/env.go

```diff
@@ -103,74 +106,87 @@ var (
 
 		installCRDs := clientutil.GetEnvOrDefault("TEST_INSTALL_CRDS", "true") == "true"
 		installWebhooks := len(allRegisterWebhookFuncs) > 0 && os.Getenv("TEST_INSTALL_WEBHOOKS") != "false"
+		if useExistingGreenhouseCluster {
+			// we are making use of https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/envtest#pkg-constants to prevent starting a new control plane
+			os.Setenv("USE_EXISTING_CLUSTER", "true")
+			kubeconfig := os.Getenv("KUBECONFIG")
+			fmt.Printf("using existing cluster with kubeconfig: %s\n", kubeconfig)
```

Comment:
Let's use the logging library for this.

Reply:
Logging is swallowed by `go test` and `Ginkgo. Therefore we need to print important test information out.

Reply:
I would like to always see this output as a reference to where the tests are running. Does that make sense?

Same as in here, where I can just copy+paste the dev-env kubeconfig for debugging....